src/sections/FAQSection/FAQSection.tsx

```diff
@@ -116,57 +132,52 @@ export const FAQSection = () => {
 
   return (
     <SectionLayout alternativeBackground>
-      <Container>
-        <Grid spacing={3} container>
-          <Grid xs={12} lg={6} item>
-            <MaxWidth>
-              <UnifiedHeading level={2} size="lg" isStrong>
-                Frequently asked questions
-              </UnifiedHeading>
-            </MaxWidth>
-          </Grid>
-          <Grid xs={12} lg={6} item>
-            {items.map(({ question, answer }, index) => {
-              const isExpanded = expanded === `panel${index}`;
-
-              return (
-                <StyledPanel
-                  key={`panel${index}`}
-                  expanded={isExpanded}
-                  onChange={handleChange(`panel${index}`)}
-                  lastChild={items.length === index + 1}
-                  square
-                >
-                  <StyledSummary aria-controls={`panel${index}d-content`}>
-                    <Topic>{question}</Topic>
-                    <StyledPlusIcon isExpanded={isExpanded} />
-                  </StyledSummary>
-                  <StyledDetails>
-                    {Array.isArray(answer) ? (
-                      <List>
-                        {answer.map(({ content, list }) => (
-                          <ListItem key={content}>
-                            {content}
-                            {Array.isArray(list) && (
-                              <List>
-                                {list.map(text => (
-                                  <ListItem key={text}>{text}</ListItem>
-                                ))}
-                              </List>
-                            )}
-                          </ListItem>
-                        ))}
-                      </List>
-                    ) : (
-                      <p>{answer}</p>
-                    )}
-                  </StyledDetails>
-                </StyledPanel>
-              );
-            })}
-          </Grid>
-        </Grid>
-      </Container>
+      <FAQContainer>
+        <FAQTitleContainer>
+          <UnifiedHeading marginBottom={70} level={2} size="lg" isStrong>
+            Frequently asked questions
+          </UnifiedHeading>
+        </FAQTitleContainer>
+        <MaxWidth>
+          {items.map(({ question, answer }, index) => {
+            const isExpanded = expanded === `panel${index}`;
+            return (
+              <StyledPanel
+                key={`panel${index}`}
```

Comment:
```suggestion
                key={question}
```

I know it's the same as before, but as we're here let's fix it. Saying key is `string + index` is the same as only `index`